### PR TITLE
fix: updated and consolidated quotes

### LINKED ISSUE #38200: Update congratulatory quotes
Refers changes from #37952

The changes from the linked PR should go into the correct file, I believe there are multiple such files.

We need to audit and combine all of them to one, and at the same time move them into the common config location.

### LINKED ISSUE #39072: Winston churchill quote not fixed
On this fine morning, Im in a good study mood and I see this yet again 
winston

### PR DESCRIPTION
**Pull request by rdm100**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #38200 
Closes #39072

### CODE REVIEW

**Review by raisedadead — CHANGES_REQUESTED** [review]
image

We have a dedicated document on what is the best practice for opening PRs.

Again with no disrespect, since this is a fresh PR, please squash all the commits to one to save everyone's time (this was the reason we had to close the original PR in the first place).

**Comment by rdm100** [comment]
@raisedadead I have squashed the commits down to 1. The code is the same, but the Travis build tests have failed even though they passed before I squashed the commits.

How do I fix this?

**Comment by rdm100** [comment]
@raisedadead fixed it, the first time I squashed the commits the last commit was left out, so I made the commit again then squashed the 2 commits into 1.

File: client/src/components/FourOhFour/index.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import Spinner from 'react-spinkit';
 import { Link } from 'gatsby';
 
 import notFoundLogo from '../../assets/images/freeCodeCamp-404.svg';
-import { quotes } from '../../resources/quotes.json';
+import words from '../../../../config/motivationalQuotes.json';
```
Comment (ojeytonwilliams) [inline client/src/components/FourOhFour/index.js:7]:
```suggestion
import words from '../../../../config/motivational-quotes.json';
```
This is a more standard filename convention.

File: client/src/utils/get-words.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import words from './words.json';
+import words from '../../../config/motivationalQuotes.json';
```
Comment (ojeytonwilliams) [inline client/src/utils/get-words.js:1]:
```suggestion
import words from '../../../config/motivational-quotes.json';
```

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
Hi @rdm100 we're nearly there, but I noticed a few things we could clean up.

Ideally filenames should be lower case, so `motivationalQuotes.json` should become `motivational-quotes.json`.

`get-words.js` appears twice, so let's remove `client/src/templates/Challenges/utils/get-words.js` in favour of `client/src/utils/get-words.js`.  Also, `randomPhrase` and `randomVerb` are no longer used, so those functions and the corresponding parts of `motivational-quotes.json` (`phrases` and `verbs`) should go.

**Comment by rdm100** [comment]
@Sky020 I removed client/src/templates/Challenges/utils/get-words.js, but had to add it back in as the build failed because client/src/templates/Challenges/redux/current-challenge-saga.js uses it.

**Comment by rdm100** [comment]
@Sky020 just realised that file should just reference the only get-words.js file remaining, so did that.

**Comment by Sky020** [comment]
That looks fine, so far. If you have done what _ojeytonwilliams_ suggested, then you can mark the request as _Resolved_. This lets him know you have made the changes, and he can review it again.

**Comment by rdm100** [comment]
@ojeytonwilliams resolved the issues you flagged I think, so it should be ready for review again.

**Review by ojeytonwilliams — APPROVED** [review]
Thanks for making those changes, @rdm100!  I made a mistake and missed some duplicated code in my first review, so I pushed a commit fixing that.

It looks good to me, now!

**Review by Manish-Giri — APPROVED** [review]
LGTM 💯

**Review by raisedadead — APPROVED** [review]

**Comment by rdm100** [comment]
cool, cheers everyone for answering any questions I had.

**Comment by ojeytonwilliams** [comment]
No problem @rdm100, thanks for taking the time to do this.  It will definitely make maintaining this a lot easier.

**Pull request merged**